client/src/components/Ticketing/ticketingmanager/Event/components/EventShowingView.tsx

```diff
@@ -24,6 +32,35 @@ export const EventShowingView = (props: EventInstanceViewProps) => {
       .slice(0, 8)}`,
   );
 
+  const onCloneSuccess = async (res) => {
+    const data = await res.json();
+    setReloadShowing((reload) => !reload);
+    setPopUpProps(
+      'Success',
+      'Showing successfully cloned',
+      true,
+      `update-modal-showing-id-${data.eventinstanceid}`,
+    );
+    setEditing((editing) => !editing);
+  };
+  const onCloneError = async (res) => {
+      const data = await res.json();
+      setReloadShowing((reload) => !reload);
+      setPopUpProps(
+          'Failure',
+          'Showing clone failed',
+          false,
+          `update-modal-showing-id-${data.eventinstanceid}`,
```

Comment:
Same here, but also if it errors, then does it return an `eventinstanceid`? If not, then this could just be a `clone-modal-failure` or something like that.

Reply:
Fixed